src/Application/Common/Extensions/EnumerableExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace CleanArchitecture.Application.Common.Extensions
+{
+    public static class EnumerableExtensions
+    {
+        public static IEnumerable<T> DeepClone<T>(this IEnumerable<T> list)
+        {
+            string serializedData = JsonConvert.SerializeObject(list);
+            var newList = JsonConvert.DeserializeObject<IEnumerable<T>>(serializedData);
+            return newList;
+        }
+    }
+}
```

Comment:
[Doubt] Is the application project the correct place for this extension as you are only using the code for testing purposes? Is it better to move this code to Application if someday we will use the DeepClone method there? I had this thought and I would like to know other points of view.

Reply:
Hello! 

Yes, I have introduced it there because it is very generic and can be used in the future to easily deeply clone any IEnumerable.

Of course, I can keep it in the test project only. I am not sure where these utility functions should stay in this solution.

Reply:
I have moved EnumerableExtensions into Application.UnitTests project, so that all the changes are now done only to this project.